guides/common/modules/con_using-pxe-to-provision-hosts.adoc

```diff
@@ -1,10 +1,11 @@
 [id="Using_PXE_to_Provision_Hosts_{context}"]
 = Using PXE to provision hosts
 
-You can provision bare-metal instances with {Project} using one of the following methods:
+You can provision bare metal instances with {Project} by using one of the following methods:
```

Comment:
Actually, there is a problem with the "bare metal" Vale rules. "bare-metal" as an adjective is OK. The problem is that the rule flagging the issue is temporarily commented out. Not sure what you should do in this case.

Reply:
```suggestion
You can provision bare-metal instances with {Project} by using one of the following methods:
```
I'll keep the hyphen and ignore Vale.